x-pack/docs/en/security/es-security-principles.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[es-security-principles]]
+== {es} security principles
+Protecting your {es} cluster and the data it contains is of utmost importance.
+Implementing a defense in depth strategy provides multiple layers of security
+to help safeguard your system. The following principles provide a foundation
+for running {es} in a secure manner that helps to mitigate attacks on your
+system at multiple levels.
+
+[discrete]
+[[security-run-with-security]]
+=== Run {es} with security enabled
+Never run an {es} cluster without security enabled. This principle cannot be
+overstated. Running {es} without security leaves your cluster exposed to anyone
+who can send network traffic to {es}, permitting these individuals to download,
+modify, or delete any data in your cluster.
+<<configuring-stack-security,Start the {stack} with security enabled>> or
+<<manually-configure-security,manually configure security>> to prevent
+unauthorized access to your clusters and ensure that internode communication
+is secure. 
+
+[discrete]
+[[security-not-root-user]]
+=== Run {es} with a dedicated non-root user
+Never run {es} as the `root` user. Running as the `root` user sheds a layer of
+security in your defense in depth strategy and permits a user to do *anything*
+on your server. {es} will refuse to start if it detects that it is running as
+`root`, but it's worth double (or triple) checking the user that {es} runs as.
+
+Instead, create a dedicated, unprivileged user to run {es} instead of running
+as the `root` user. By default, the `rpm`, `tar.gz`, `docker`, and Windows
+distributions come prepackaged with an `elasticsearch` user.
```

Comment:
```suggestion
Never try to run {es} as the `root` user, as this would make any defense strategy not effective and permits a malicious user to do anything on your server.

You must create a dedicated unprivileged user to run {es}. By default, the `rpm`, `deb`, `docker`, and Windows packages come with an `elasticsearch` user ready to use for this scope.
```